examples/tabs/palette-exemplar/package.json

```diff
@@ -87,7 +86,8 @@
     "webpack-cli": "^3.2.1",
     "webpack-bundle-analyzer": "^3.0.3",
     "webpack-dev-server": "^3.1.14",
-    "webpack-merge": "^4.2.1"
+    "webpack-merge": "^4.2.1",
+    "@misk/cli": "^0.1.5-2"
```

Comment:
We'll have to remove it from `@misk/cli/utils/templates/package.ts` and publish the CLI or else this will keep showing up.